staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/types.go

```diff
@@ -67,5 +69,24 @@ type Key struct {
 	Secret string `json:"secret"`
 }
 
+// GKMSConfig contains API configuration for Google KMS transformer.
+type GKMSConfig struct {
+	// projectID is the GCP project which hosts the key to be used. It is automatically set
+	// to your project if you are running Kubernetes on GKE/GCE.
+	// +optional
+	ProjectID string `json:"projectID,omitempty"`
+	// location is the location of the hosted key to be used. The default value is "global".
```

Comment:
What does location mean? The region or zone where the key is located? Is it the location of the key or the key ring?